pure.zsh

```diff
@@ -57,17 +59,28 @@ prompt_pure_clear_screen() {
 }
 
 prompt_pure_check_git_arrows() {
+	# reset git arrows
+	prompt_pure_git_arrows=
```

Comment:
That's actually the point of this PR: if we return a value, then we need to run this function in a subshell and capture the output with command substitution; however, if we save the result in a global variable, we avoid that subshell, i.e. forking.